### ISSUE TITLE
Add YAML Frontmater to pages. 

### ISSUE THREAD

**Issue by danhannigan** [Denver-Devs/denverdevs.org#7]
Add title, meta, etc to individual pages - see this page for reference: https://vuepress.vuejs.org/guide/markdown.html#front-matter

**Comment by VonTeacher** [comment]
I'd like to help with this. I've got it running locally and successfully replaced page titles with front matter. When you mention "meta, etc." what else are you thinking? @vew on DD Slack.

**Comment by danhannigan** [comment]
Hey @VonTeacher 👋 

Thanks for taking a look! When I mention "meta" and the like I'm referring to the metadata content you'd typically see in a well-defined <head> section of a web-page. https://vuepress.vuejs.org/guide/frontmatter.html#predefined-variables --> and more specifically: https://vuepress.vuejs.org/guide/frontmatter.html#meta. 

These are bits of information that are more-so helpful for SEO / and some accessibility and don't have any impact on the page performance so they're not usually a high-priority to most. They're good practice, though, and should be specific to each page - right now we just have generic variables for all pages. If you're totally unfamiliar check out: https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/The_head_metadata_in_HTML

Basically I'm asking for custom metadata to be plugged in for each page. Possibly even Open Graph Data for social sharing and such as well. Since this is a somewhat large task (in execution, not in difficulty) that would require some clever copyrighting we can break it up into subtasks if need be. 

Let me know if you want to proceed :)

**Comment by VonTeacher** [comment]
Thanks for the great information. If it's alright with you, I wouldn't mind attempting to tackle a small bit (start with one page?) and then re-evaluate after that, both for my own check on ability in addition to seeing if it's the kind of work that's being sought.

Let me know what next steps might be. Cheers!

**Comment by danhannigan** [comment]
Happy to provide any info I can! Feel free to pluck off any page that seems interesting to you and take a crack at it. 

Next steps would be to follow typical contribution procedure - fork, do some work, pop up a PR and so on. If you're a bit fuzzy on that you can check out this pretty succinct write-up https://gist.github.com/MarcDiethelm/7303312 (and this isn't me assuming you don't know, I assume you do! But I'd rather over-inform than under-inform just in case). 

At a minimum what I'd say you could shoot for is a custom description meta field for the page. If you want to confirm it builds right - after it compiles you can pop open the inspector on the page and view the <head> element - your updated meta should be living in there!

**Comment by VonTeacher** [comment]
This is really helpful. Thanks. I'll start this weekend and check back in.
